src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/Stage.java

```diff
@@ -87,6 +87,10 @@
     @Column(name = "case_created", insertable = false, updatable = false)
     private LocalDateTime caseCreated;
 
+    @Getter
+    @Column(name = "text", insertable = false, updatable = false)
```

Comment:
Is text a good name for this? could we make this a little clearer with an alias?

Reply:
Updated with alias